wandb/docker/__init__.py

```diff
@@ -211,6 +212,23 @@ def get_image_uid(image_name: str) -> int:
     return int(shell(["run", image_name, "id", "-u"]))
 
 
+def push(ecr_string: str, tag: str) -> Optional[str]:
+    """Push an image to an ECR repository"""
```

Comment:
could you make this function generalized, eg `repo` instead of `ecr_string`, it's the same thing for all registries

Reply:
Nice yeah